core/modules/wdb/QueryRunner.cc

```diff
@@ -276,7 +279,7 @@ class ChunkResourceRequest {
         : _mgr{mgr}, _msg{msg} {}
 
     ChunkResource getResourceFragment(int i) {
-        wbase::Task::Fragment const& f(_msg.fragment(i));
+        auto const& f(_msg.fragment(i));
```

Comment:
I've seen this used several times. I guess I am mystified. The deleted code made some sense because I know the type. The liberal use of auto hides all of that (yes, I am not a fan of over-using auto). Also, I would use something other than single letter objects as it's too easy to loose track of what 'f' really is.